pkg/codegen/internal/test/testdata/simple-enum-schema/docs/tree/v1/rubbertree.md

```diff
@@ -623,7 +623,7 @@ The following state arguments are supported:
 <a href="#brightness_go" style="color: inherit; text-decoration: inherit;">Brightness</a>
 </span>
         <span class="property-indicator"></span>
-        <span class="property-type"><a href="#containerbrightness">Container<wbr>Brightness</a></span>
+        <span class="property-type"><a href="#containerbrightness">float64</a></span>
```

Comment:
Shouldn't this remain `ContainerBrightness`?

Reply:
Same answer as above.